lint-staged.config.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable filenames/match-regex */
+const escapedFileNames = (filenames) =>
+  filenames.map((filename) => `"${filename}"`).join(' ');
+
+module.exports = {
+  './*.{js,jsx,ts,tsx}': (files) => {
```

Comment:
A bit much on the extensions, can 100% remove `jsx|tsx` however would like to move project to TypeScript. So can stay for now and be removed later.